tests/initial_test.sh

```diff
@@ -1,22 +1,24 @@
+#!/bin/sh
+
 # Simple regression test for forward solver
 # Should only be run using default settings, when first installed
 #
 # Currently testing by confirming generated file sizes are the same.
 # This is a fairly crude method, but md5 hashing won't work, due to slightly different (at >10 decimal places)
 # voltages/jacobians being generated on different processers.
 
-#! /bin/bash
```

Comment:
:joy: this was really bugging you :wink:

Reply:
That was useless!